core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -884,7 +890,7 @@ public void testCreateTableWithDefaultColumnValue() {
 
     catalog
         .buildTable(ident, schemaWithDefault)
-        .withLocation("file:/tmp/ns/table")
+        .withLocation(baseTableLocation(TBL))
```

Comment:
```suggestion
        .withLocation(baseTableLocation(ident))
```
this one should be referring to the identifier local to the test

Reply:
I changed ident with TBL too